__integrations__/models/message.test.ts

```diff
@@ -242,4 +262,290 @@ describe("Models |> Message", () => {
+      // check that every message has a createdDate less than the previous one
+      if (i.resource.isRight()) {
+        expect(i.resource.value.createdAt < prevMessageCreationDate).toBe(true);
+        prevMessageCreationDate = i.resource.value.createdAt;
+      }
+    });
+  });
+
+  it("should retrieve only one page without any continuation token when the pageSize is greater than the number of results", async () => {
+    const context = await createContext(MESSAGE_MODEL_PK_FIELD);
+    await context.init();
+    const model = new MessageModel(context.container, MESSAGE_CONTAINER_NAME);
+
+    await createMessage(model, 1);
+    await createMessage(model, 2);
+
+    // get a page of messages by fiscal code
+    let results: DecodedFeedResponse<RetrievedMessage> = await model
+      .findMessages(aFiscalCode, 10 as NonNegativeInteger)
+      .map((ai: AsyncIterator<DecodedFeedResponse<RetrievedMessage>>) =>
+        ai.next().then(ir => ir.value)
+      )
+      .getOrElseL(_ => {
+        throw new Error("Error");
+      })
+      .run();
+
+    expect(results.length).toEqual(2);
+    let prevMessageCreationDate = new Date();
+    results.forEach(i => {
+      expect(i.hasMoreResults).toBe(false);
+      expect(i.continuationToken).toBe(undefined);
+      expect(i.resource.isRight()).toBe(true);
+      // check that every message has a createdDate less than the previous one
+      if (i.resource.isRight()) {
+        expect(i.resource.value.createdAt < prevMessageCreationDate).toBe(true);
+        prevMessageCreationDate = i.resource.value.createdAt;
+      }
+    });
+  });
+
+  it("should retrieve only one page with a continuation token when the pageSize is less than the number of results", async () => {
+    const context = await createContext(MESSAGE_MODEL_PK_FIELD);
+    await context.init();
+    const model = new MessageModel(context.container, MESSAGE_CONTAINER_NAME);
+
+    await createMessage(model, 1);
+    await createMessage(model, 2);
+    await createMessage(model, 3);
+
+    // get a page of messages by fiscal code
+    let results: DecodedFeedResponse<RetrievedMessage> = await model
+      .findMessages(aFiscalCode, 2 as NonNegativeInteger)
+      .map((ai: AsyncIterator<DecodedFeedResponse<RetrievedMessage>>) =>
+        ai.next().then(ir => ir.value)
+      )
+      .getOrElseL(_ => {
+        throw new Error("Error");
+      })
+      .run();
+
+    expect(results.length).toEqual(2);
+    let prevMessageCreationDate = new Date();
+    results.forEach(i => {
+      expect(i.hasMoreResults).toBe(true);
+      expect(i.continuationToken).not.toBe(undefined);
+      expect(i.resource.isRight()).toBe(true);
+      // check that every message has a createdDate less than the previous one
+      if (i.resource.isRight()) {
+        expect(i.resource.value.createdAt < prevMessageCreationDate).toBe(true);
+        prevMessageCreationDate = i.resource.value.createdAt;
+      }
+    });
+  });
```

Comment:
We must manually dispose the created container after the test ends
```suggestion
    await context.dispose();
  });
```

Reply:
Done